packaging/windows/BUILD.txt

```diff
@@ -0,0 +1,120 @@
+How to make a darktable windows installer (64 bit only):
+
+1). Install MSYS2 (instructions and prerequisites can be found on official website: https://msys2.github.io/).
+    
+    Install x64 developer toolchain from the MSYS2 prompt:
+    $ pacman -S mingw-w64-x86_64-toolchain
+    
+    Install required dependencies:
+    $ pacman -S mingw-w64-x86_64-{exiv2,lcms2,lensfun,libglade,dbus-glib,libusb-compat,openexr,sqlite3,libxslt,libsoup,libwebp,libsecret,lua,graphicsmagick,openjpeg2,gtk-engines,gtk3,pugixml,sdl,libexif}
+    
+    For osm-gps-map:
+    $ pacman -S mingw-w64-x86_64-{python2-cairo,python2-pygtk,osm-gps-map}
+
+    For gphoto2:
+    $ pacman -S mingw-w64-x86_64-{libusb,popt,libgphoto2}
+    Check that CAMLIBS and IOLIBS for LIBGPHOTO are available
+
+    For Saxon:
+    $ pacman -S unzip perl-Locale-Gettext perl-TermReadKey perl-sgmls
+
+    For usermanual 
+    $ pacman -S mingw-w64-x86_64-imagemagick
+
+    For other gtk libraries:
+    $ pacman -S mingw-w64-x86_64-{atkmm, glibmm, cairomm, gtkmm3, gtksourceview3}
+
+
+    Install SAXON:
+    https://sourceforge.net/projects/saxon/files/Saxon-HE/
+    copy files into /usr/share/java/saxon/
+
+    Install DocBook Extension to Saxon 
+    https://sourceforge.net/projects/docbook/files/docbook-xsl-saxon/1.00/
+    copy files to /usr/share/java/docbook-xsl-saxon
+
+    Install FOP
+    http://www.apache.org/dyn/closer.cgi/xmlgraphics/fop
+    copy files to /usr/share/java/fop
+    copy fop.xconf to /etc/fop.conf
+
+    Install PO4A:
+    https://alioth.debian.org/frs/?group_id=30267
+    Extract and copy to your home folder
+    $ perl Build.PL
+    $ ./Build
+    $ ./Build install
+
+    Modify your bash_profile file in your HOME directory
+    # added as per http://wiki.gimp.org/wiki/Hacking:Building/Windows
+    export PREFIX="/mingw64"
+    export LD_LIBRARY_PATH="$PREFIX/lib:$LD_LIBRARY_PATH"
+
+    JAVA_HOME="/C/Program Files (x86)/Java/jre1.8.0_101"
```

Comment:
This looks like you are compiling natively on Windows and not cross compiling? That should be mentioned at the very top of the file.